common/app/common/NewNavigation.scala

```diff
@@ -379,16 +379,19 @@ object NewNavigation {
       }
     }
 
-    def getTopLevelSection(id: String) = {
-      val sectionList = sectionLinks.filter { item =>
-        item.pageId == id
-      }
+    def getSectionLink(id: String): Option[String] = {
+      sectionLinks.find(_.pageId == id).map(_.parentSection.name)
+    }
```

Comment:
Are `getSectionLinks` and `getSectionLink` only supposed to use internally? If yes make them `private`